submission/proxy.go

```diff
@@ -157,6 +157,17 @@ func (p *Proxy) restartDistributor(ctx context.Context, ll *loglist.LogList) err
 	return nil
 }
 
+// Refresh runs standalone Log-list refresh request.
+func (p *Proxy) Refresh(ctx context.Context) {
+	p.distMu.Lock()
+	defer p.distMu.Unlock()
+	if p.dist == nil {
+		p.Errors <- fmt.Errorf("Calling Refresh() on non-initialized proxy, call Run() first")
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Also, I think it's possible to call Run(), then call Refresh() and still get this error. If the gorountine started by Run() hasn't received a loglist update yet, it won't have called `p.restartDistributor()`, so therefore won't have initialized `p.dist` yet. Isn't asynchronicity fun? :wink:

Reply:
Why do you need `p.dist != nil` at this point?

Reply:
Also, apparently this method is never actually used in the tests.

Reply:
Removed this method.